app/pipelines/routes.py

```diff
@@ -140,17 +146,16 @@ def pipeline_search():
     # else take the first one and set logo
     # TODO right now, this handles CBRAIN and one other platform
 
-    #Mandana issue 378
+    # Mandana issue 378
```

Comment:
```suggestion
    # read the list of pipelines also available in CBRAIN
```

Reply:
the comment above was not clear. I know this is not technically part of your PR but it might be good to update it to a more useful comment anyway since you are changing the line.